src/plugins/cascade/src/main/js/core/CustomStyleFormatsUtils.js

```diff
@@ -112,11 +226,39 @@ define(
         return uniqueImageClasses;
       };
 
+      var getFormatInlineStyles = function (selectedCustomFormatNames, customStyleFormats) {
```

Comment:
I originally added logic for merging inline styles with any existing inline styles but ran into a lot of issues getting it working properly & the logic was getting more complex & time-consuming on top of what was already added. 

Mainly, it was hard to determine if an inline style was existing previously because for some styles, i.e. something like `border: 1px solid green` would automatically get converted into a separate style, 'border-width: 1px'; when the dialog was submitted; so simply comparing to the format "combined" version wouldn't work. Wondering if there's a more convenient way to do this, or possibly tabling it for now & considering in post issue (hopefully, there isn't a heavy reliance on inline styling).

Reply:
This was reworked in 270813b